### DISCUSSION TITLE
Export/Import/Transfer Partial data

### DISCUSSION THREAD

**Discussion by Falci** [dbeaver/dbeaver#34262] [Q&A] [answered]
Hello,

I have a quite big table in production and I'd like to copy some data to staging/test environment. But I don't need the entire table. Is there a way to set a query to limit which data should be imported?

**Comment by TB-HunBoka** [comment]
Hello,

I don't know which database do you use, but I think there are multiple ways of doing so.
1. Export the required data in to an SCV object, and simple insert it back to the dev system.
2. If it is realy some rows, then select the data you would like to insert, right click, advanced copy, SQL. This will generate an insert command for you. Obviously you need to create the table first.

**Reply by Falci** [reply, marked as answer]
Thanks.

The problem was: I was trying to export from the sidebar, clicking in the table name. From there there's no way to add filters.

The correct approach is: open the SQL editor, make and run the query, then click in the export below the result. From there we can export to another table
